src/libraries/System.Private.CoreLib/src/Resources/Strings.resx

```diff
@@ -3972,7 +4243,7 @@
     <value>Type names passed to Assembly.GetType() must not specify an assembly.</value>
   </data>
   <data name="NoMetadataTokenAvailable" xml:space="preserve">
-    <value>There is no metadata token available for the given member.</value>
+    <value>There is no metadata token available for the given member, MemberImplementationType: {0}.</value>
```

Comment:
```suggestion
    <value>There is no metadata token available for the given member.</value>
```
The existing uses of this string would produce bad messages with this change. I do not think it is particularly interesting to include `MemberImplementationType` in the error message - the error message seems to be unreachable code.